src/overlays/actors/ovl_En_Minideath/z_en_minideath.c

```diff
@@ -835,7 +835,7 @@ void EnMinideath_Update(Actor* thisx, PlayState* play) {
         Actor_SetFocus(&this->actor, 0.0f);
 
         effect = this->effects;
-        elem = this->collider.elements;
+        jntSphElem = this->collider.elements;
```

Comment:
```suggestion
        jntSphElem = &this->collider.elements[0];
```
Prayge surely it matches